API_DOCS.md

```diff
@@ -579,6 +581,34 @@ Get all of the survey data for a given exhibit.
 
 You must be authorized as an admin to hit this endpoint.
 
+
+## `POST /admin/feature/{id}`
+
+Marks an exhibit as featured.
+If the exhibit is already featured, this will have no effect.
+
+### Authorization
+
+You must be authorized as an admin to hit this endpoint.
+
+### Response
```

Comment:
Please use the same format as everything else? The response codes, because they're universal, are already described at the very top. The `### Response body` sections describe what's in the body of the response. Look at how the support endpoints are documented.

Reply:
Done